### PR TITLE
chore: Bump Dependencies

### PR DESCRIPTION
**Pull request by kjellhaaland**
### Short summary
Updated most dependencies to latest version.
EDS and Cognite/3D packages are not updated.
Added script for deps update in package.json
Updated all usages of useQuery to latest version.

Link to issue: https://github.com/equinor/lighthouse/issues/554

### PR Checklist
- [x] I have performed a self-review of my own code
- [x] I have written a short summary of my changes in the PR
- [x] I have linked related issue to the PR

> [!TIP]
> To deploy the PR to Test, use the [Manual deploy fusion app TEST🚀](https://github.com/equinor/cc-components/actions/workflows/manual-deploy.yml) action.
> Remember to deploy any backend changes to Test as well!

> [!CAUTION]
> ⛔ I understand by merging my PR, the changes will be deployed to production immediately ⛔

### CODE REVIEW

File: libs/shared/package.json
Comment (kjellhaaland) [inline libs/shared/package.json:1]: workspace-ag-grid must be updated to latest fusion-workspace before deployment

File: libs/heattracesidesheet/src/lib/utils-sidesheet/useGetWorkorders.ts [hunk 1]
```diff
@@ -4,9 +4,10 @@ import { useQuery } from '@tanstack/react-query';
 export const useGetWorkorders = (heatTraceCabelId: string) => {
```
Comment (Gustav-Eikaas) [inline libs/heattracesidesheet/src/lib/utils-sidesheet/useGetWorkorders.ts:4]: typo

File: libs/shared/package.json [hunk 2]
```diff
@@ -88,17 +88,17 @@
...
   "dependencies": {
     "@cc-components/sharedcomponents": "workspace:^",
-    "@emotion/react": "^11.11.1",
-    "@emotion/styled": "^11.11.0",
-    "@equinor/fusion-react-person": "^0.8.0",
-    "@equinor/workspace-ag-grid": "2.0.5",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@equinor/fusion-react-person": "^0.8.5",
+    "@equinor/workspace-ag-grid": "3.0.1-pr.0",
```
Comment (Gustav-Eikaas) [inline libs/shared/package.json:94]: Publish before merging

File: libs/workordersidesheet/src/lib/utils-sidesheet/useMccr.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ export const useMccr = (packageId: string | null) => {
   const { data, isFetching, error } = useQuery({
     queryFn: (a) => fetchMccr(ccApp, contextId, packageId, a.signal),
     queryKey: ['mccr', packageId],
+    initialData: [],
```
Comment (Gustav-Eikaas) [inline libs/workordersidesheet/src/lib/utils-sidesheet/useMccr.ts:32]: Are you sure this is the appropriate solution?

**Review by Gustav-Eikaas — APPROVED** [review]
Great work Kjell!

File: libs/heattracesidesheet/src/lib/utils-sidesheet/useGetWorkorders.ts (hunk 1 above)
Comment (kjellhaaland) [reply libs/heattracesidesheet/src/lib/utils-sidesheet/useGetWorkorders.ts:4]: Fixed typo

File: libs/workordersidesheet/src/lib/utils-sidesheet/useMccr.ts (hunk 3 above)
Comment (kjellhaaland) [reply libs/workordersidesheet/src/lib/utils-sidesheet/useMccr.ts:32]: Removed this.

**Pull request merged**